core/state/statedb.go

```diff
@@ -488,7 +488,7 @@ func (self *StateDB) Copy() *StateDB {
 func (self *StateDB) Snapshot() int {
 	id := self.nextRevisionId
 	self.nextRevisionId++
-	self.validRevisions = append(self.validRevisions, revision{id, self.journal.Length()})
+	self.validRevisions = append(self.validRevisions, revision{id, len(self.journal.entries)})
```

Comment:
Not sure why you removed `Length` ? Is this really cleaner?

Reply:
probably not